biz.aQute.bnd.gradle/src/aQute/bnd/gradle/Export.groovy

```diff
@@ -208,7 +209,16 @@ public class Export extends DefaultTask {
         throw new GradleException("${run.getPropertiesFile()} workspace errors")
       }
 
+      def parent = run.parent;
       try {
+        if (parent instanceof Processor) {
+          Properties gradleProperties = new PropertiesWrapper()
+          gradleProperties.put('task', this)
+          gradleProperties.put('project', project)
+          Processor processor = new ProcessorWrapper(gradleProperties)
```

Comment:
Why do you need ProcessorWrapper? Isn't `new Processor(gradleProperties, false)` enough?

Reply:
The default `setParent` is not without side effects. In that implementation the properties of the `parent` are merged with those of the child. This breaks the nature of `PropertiesWrapper` which, has no real properties, is designed to only have effect when called directly. So this change is absolutely necessary to preserve that behavior.